src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -87,17 +87,33 @@ angular.module("ovh-angular-mondial-relay")
                 });
             };
 
-            var result = _.chain(MONDIAL_RELAY.weekDays)
-                .filter(function (weekDay) {
-                    return !!opening[weekDay];
-                })
-                .map(function (weekDay) {
-                    return {
-                        day: weekDay,
-                        opening: getAllOpenings(opening[weekDay])
-                    };
-                })
-                .value();
+            var result = MONDIAL_RELAY.weekDays.map(function (weekDay) {
```

Comment:
No ES6 ?

Reply:
Whats a result ? It looks like it should be `openingTimeWindows`

Reply:
ES6 is not embedded, don't know why

Reply:
Currently Babel is not embedded.
(see https://github.com/ovh-ux/ovh-angular-mondial-relay/blob/master/package.json)
it's a new feature to develop, if needed.